### ISSUE TITLE
I can't open the Dashboard on Interative Learning Portal

### ISSUE THREAD

**Issue by hhidd0211** [openshift-labs/learn-katacoda#603]
The terminal is ok, I can connect and log in. But I can't open the web portal.
![Snip20191030_1](https://user-images.githubusercontent.com/18681621/67833576-fc7b2e80-fb1f-11e9-9626-32c79aadd423.png)

**Comment by BenHall** [comment]
Hi @hhidd0211 

Are you based within China? Given the error, I feel the IP might be blocked as we run on Google's LB. 

Please confirm and we'll investigate if there are other ways we can manage traffic.

**Comment by hhidd0211** [comment]
Yes, I'm based in China and tried to access&nbsp;the web console from my company's network in China. I can access Google in this network.



发自我的iPhone


------------------ Original ------------------
